.github/ISSUE_TEMPLATE/support-request.md

```diff
@@ -1,13 +1,13 @@
 ---
 name: Support request
-about: If you need help
+about: If you need additional help
 title: "[Support]"
 labels: ''
 assignees: ''
 
 ---
 
-_Please read the "Discussion / Support" section of the Readme first. Only open an issue if you don't get an answer on the OBS or someone tells you to. _
+_Please read the "Discussion / Support" section of the Readme first. Only open an issue if you don't get an answer on the OBS or someone tells you to._
```

Comment:
probably typo. You mean `on the OBS discord` right?

Reply:
Oh damnit.
Yes I did.